include/translate-option.php

```
@@ -194,7 +194,7 @@ public function pre_update_option( $value, $old_value, $name ) {
 		// Stores the unfiltered old option value before it is updated in DB.
 		remove_filter( 'option_' . $name, array( $this, 'translate' ) );
 		$unfiltered_old_value = get_option( $name );
-		add_filter( 'option_' . $name, array( $this, 'translate' ), 20, 2 );
+		add_filter( 'option_' . $name, array( $this, 'translate' ), 20 );
```

Please temporarily ignore the error. This line will be removed by #1111